### ISSUE TITLE
Remove big modules like request/handlebars

### ISSUE THREAD

**Issue by shmuga** [Travelport-Ukraine/uapi-json#111]

**Comment by Smotrov** [comment]
Please try to use native requests like this. 
```
var http = require('http');
http.request(options, callback).end();
```
If it supports uAPI authentication. 
Having 4Mb+ code for doing basic HTTPS requests is too much as for me.

**Comment by dchertousov** [comment]
@shmuga I'd like to keep it simple too, but I'd like more complicated things like redirects etc to be handled by 3rd party. Would you mind to change `request` to `axios`? It has promise-based API and a lot smaller footprint.

**Comment by shmuga** [comment]
Yes, I like this module. You can easily use it.

**Comment by dchertousov** [comment]
As for `handlebars`, it's the best mustache-like engine, but to minimize footprint we could replace it with `doT.js`.

@shmuga what do you think?

**Comment by shmuga** [comment]
I think we should not hurry up with replacing template engine as we have no tests for templates. This can cause a lot of errors and so on. 
I think we keep `handlebars` now.

**Comment by dchertousov** [comment]
Closed by #149
